pyintegration/deephaven2/uri.py

```diff
@@ -0,0 +1,87 @@
+#
+#   Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+""" Tools for resolving URIs into objects including Tables. """
```

Comment:
```suggestion
""" Tools for resolving Uniform Resource Identifiers (URIs) into objects. """
```
I'm not sure we want to be too specific here.